src/extension.ts

```diff
@@ -123,13 +129,13 @@ function executeVercorsCommand() {
         outputChannel.appendLine(data.toString());
     });
 
-    vercorsProcess.on('exit', function() {
+    vercorsProcess.on('exit', function () {
         vercorsProcessPid = -1;
-      })
+    });
 
       
     // Show the output channel
-    outputChannel.show(vscode.ViewColumn.Three, true); // Change the ViewColumn as needed
```

Comment:
what does this do? seems weird to just remove it

Reply:
it is deprecated and I noticed no difference. The documentation said this:
![image](https://github.com/utwente-fmt/vercors-ide/assets/54892083/4052a68a-8ebb-4eb7-814d-752a74a1ef30)